.github/workflows/deploy.yml

```diff
@@ -92,4 +92,16 @@ jobs:
           TF_VAR_alert_email_address: ${{secrets.alert_email_address}}
           TF_VAR_backoffice_environment_file: ${{ secrets.backoffice_env_file }}
           TF_VAR_webapp_environment_file: ${{ secrets.webapp_env_file }}
+          TF_VAR_production_webapp_ssl_verification_name: ${{ secrets.production_webapp_ssl_verification_name }}
+          TF_VAR_production_webapp_ssl_verification_value: ${{ secrets.production_webapp_ssl_verification_value }}
+          TF_VAR_production_backoffice_ssl_verification_name: ${{ secrets.production_backoffice_ssl_verification_name }}
+          TF_VAR_production_backoffice_ssl_verification_value: ${{ secrets.production_backoffice_ssl_verification_value }}
```

Comment:
I would hope we can infer the production values from the outputs of the terraform modules we're deploying to prod.